themes/finna2/templates/RecordDriver/DefaultRecord/document-links.phtml

```diff
@@ -39,7 +38,7 @@
         $descriptionDefault = ($document['displayAsLink'] ?? false) ? $this->translate('external_online_link') : null;
         ?>
         <a <?=$this->htmlAttributes($aAttrs)?>>
-            <?=$this->icon('external-link')?>
```

Comment:
sit jos se määriteltäis siel niin tähä vois laittaa vaa $this->icon($document['icon'] ?? eternal-link)